packages/chat-sdk/src/channel/channel.ts

```diff
@@ -41,11 +42,14 @@ export class Channel implements BaseChannel {
   private userReactionsSubscription: (() => void) | null = null;
   private channelReactionsSubscription: (() => void) | null = null;
   private reactionsErrorsSubscription: (() => void) | null = null;
+  private chatConfigListenerUnsubscribe: (() => void) | null = null;
   public markReadDebounced: () => void;
   public polls: BasePolls | null = null;
   private reactionsAPI: BaseReactionsAPI;
   private fetchPreviousMessagesPromise = false;
   private totalLimit: number | null = null;
+  private chatConfigChangesCallbacks: { [type: string]: ((item: any) => void)[] } = {};
+  private pinnedMessageId: string | null = null;
 
   private constructor(public channel: LiveChatChannel, private readonly chatRoom: ChatRoom) {
     this.watchChatConfigChanges();
```

Comment:
Let's create a method that will update `this.channel` and use that method as the first argument in the `this.watchChatConfigChanges();` call, and set the `ChatConfigType.ALL_CHAT_CHANGES` enum as a second argument.